### ISSUE TITLE
Rule 'no-useless-concat' was not found

### ISSUE THREAD

**Issue by preco21** [adametry/gulp-eslint#92]
When I using webpack with gulp-eslint, I have this warning:

```
C:\Workspace\Now Working\Test\src\index.js
  1:1  warning  Definition for rule 'no-useless-concat' was not found  no-useless-concat

✖ 1 problem (0 errors, 1 warning)
```

It's looks like the `eslint` version is not updated in `dependencies`.
Please solve this!

Related: http://eslint.org/docs/rules/no-useless-concat

**Comment by adametry** [comment]
gulp-eslint's dependency is on the latest minor version (i.e., [^1.0.0 includes 1.3.1](https://docs.npmjs.com/misc/semver#caret-ranges-1-2-3-0-2-5-0-0-4)). Reinstall your local node_modules or use `npm update gulp-eslint --depth=2`.

**Comment by preco21** [comment]
Oh thansk a lot!
Solved
